clients/privacy-center/pages/api/fides-js.ts

```diff
@@ -102,7 +102,7 @@ export default async function handler(
   res: NextApiResponse
 ) {
   // Load the configured consent options (data uses, defaults, etc.) from environment
-  const environment = await loadPrivacyCenterEnvironment();
+  const environment = await loadPrivacyCenterEnvironment({});
```

Comment:
Follow up from comment above about making this property optional...
The complexity is being surfaced to the interface here, whereas it's always best to keep complexity down inside the method.